### PR TITLE
add schema versioning, show in buildinfo

### LINKED ISSUE #2856: Document applied migrations
I propose that we store the current number of applied migrations in the database.

This could be realized through a new table with a single cell that is updated at the end of every migration: `UPDATE releaseinfos SET schemaVersion=12;`

This would make manual migrations easier and would pave the way to automated migrations.

What do you think?

### PR DESCRIPTION
**Pull request by fm3**
### URL of deployed dev instance (used for testing):
- https://schemaversion.webknossos.xyz

### Steps to test:
- open api/buildinfo
- should show schema version 16

### Issues:
- fixes #2856

------
- ~~[ ] Updated [changelog](../blob/master/CHANGELOG.md#unreleased)~~
- [x] Updated [migration guide](../blob/master/MIGRATIONS.md#unreleased) if applicable
- [x] Ready for review

### CODE REVIEW

File: conf/evolutions/016-add-schema-version.sql [hunk 1]
```diff
@@ -0,0 +1,11 @@
+-- https://github.com/scalableminds/webknossos/pull/2904
+
+-- From now on we store schemaVersion in the database
+-- When editing the schema, please update the number both in your evolution and in schema.sql
+
+START TRANSACTION;
+CREATE TABLE webknossos.releaseInformation (
+  schemaVersion BIGINT NOT NULL
```
Comment (normanrz) [inline conf/evolutions/016-add-schema-version.sql]: Should we add a timestamp of last schema change?

Comment (fm3) [reply conf/evolutions/016-add-schema-version.sql]: Not sure if the benefit is worth having to remember updating it in every evolution. What do you think, what would it be useful for?

Comment (jstriebel) [reply conf/evolutions/016-add-schema-version.sql]: using `ADD COLUMN … TIMESTAMP DEFAULT NOW()` should solve this without extra effort

Comment (fm3) [reply conf/evolutions/016-add-schema-version.sql]: sure but still you’d have to add `update releaseInformation set updatedTimestamp = NOW()` to every evolution.

Comment (normanrz) [reply conf/evolutions/016-add-schema-version.sql]: could be a trigger. anyways, not sure if useful at all.

**Pull request merged**